styles.css

```diff
@@ -1,3 +1,335 @@
-* {
-    font-size: 1.5rem;
+li a {
+    font-family: "Bakbak One", serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    list-style: none;
+    text-decoration: none;
+    color: #fff;
+}
+
+.toggle-button {
+    padding: 0;
+    border: 0;
+    height: 48px;
+    background-color: transparent;
+    cursor: pointer;
+}
+
+.discord-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    font-size: 16px;
+    color: #fff;
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 24px;
+    border: none;
+    width: 147px;
+    height: 48px;
+    background-color: #141B22;
+    cursor: pointer;
+}
+.discord-button img {
+    display: block;
+}
+
+.connect-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    font-size: 16px;
+    color: #000;
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 24px;
+    border: none;
+    height: 48px;
+    width: 149px;
+    background: #21E786;
+    cursor: pointer;
+}
+connect-button img {
+    display: block;
+}
+
+.breadcrumbs a {
+    font-size: 14px;
+    color: #000;
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    line-height: 20px;
+}
+
+h1,
+h2 {
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 56px;
+    color: #fff;
+    margin: 0;
+}
+
+h3 {
+    font-family: "Bakbak One", serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+    color: #fff;
+}
+
+p {
+    font-family: Poppins, serif;
+    font-weight: 400;
+    font-size: 18px;
+    color: #C2C3C5;
+}
+
+.list-term {
+    font-family: "Bakbak One", serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.list-description {
+    font-family: "Bakbak One", serif;
+    font-size: 56px;
+    font-weight: 400;
+    line-height: 68px;
+    color: #fff;
+    text-shadow: 0 4px 16px #FFFFFF66;
+}
+
+.list-item-number {
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    font-size: 80px;
+    line-height: 96px;
+    color: #21E786;
+    opacity: 30%;
+}
+
+.list-item-sub {
+    font-family: Poppins, serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    color: #C2C3C5;
+}
+
+body {
+    background: #0D0D11;
+    text-align: center;
+    margin: auto;
+    width: 1920px;
+    height: 3288px;
+    top: 4592px;
```

Comment:
**The coordinates here are unnecessary:**

Using coordinates (like absolute positioning with top, left, right, or bottom) can lead to unnecessary complexity, especially if they’re not needed for achieving the desired layout. In this context, the use of coordinates doesn't add value and might even complicate the design, making it harder to maintain or adapt for different screen sizes. It's better to rely on more flexible layout techniques, like flexbox or grid, unless you specifically need precise positioning.